### PR TITLE
Feature/aut 718/enable scrollable double column

### PR DESCRIPTION
**Pull request by AnastasiaArcadia**
**Related to:** https://oat-sa.atlassian.net/browse/AUT-718

**Description:** Add option to enable scrollable double-column layout.

**How to test:**

1. In `config/taoQtiItem/qtiCreator.conf.php` add:
 ```
'scrollable-multi-column' => true
 ```
2. Go to the item authoring
3. Create item
4. Open item Style Editor
5. Check `Scrollable multi-column` checkbox in `Layout` section
6. Check that `.grid-row` element contains `dual-column-layout` css class (see [comment](https://oat-sa.atlassian.net/browse/TRN-930?focusedCommentId=141949))
7. Save the item
8. Reload the page or return to manage items, etc.
9. Open the item again
10. Check that `Scrollable multi-column` checkbox is checked and css class is present

**Deployed on:** http://test-anastasia.playground.kitchen.it.taocloud.org:44451

### CODE REVIEW

**Review by Silvia-loza — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

File: controller/QtiCreator.php [hunk 1]
```diff
@@ -300,6 +300,12 @@ protected function getCreatorConfig(core_kernel_classes_Resource $item)
             $config->addPlugin('blockAdder', 'taoQtiItem/qtiCreator/plugins/content/blockAdder', 'content');
         }
 
+        if ($config->getProperty('scrollable-multi-column') == true) {
+            $config->addPlugin('layoutEditor', 'taoQtiItem/qtiCreator/plugins/panel/layoutEditor', 'panel');
+        } else {
+            $config->removePlugin('layoutEditor');
```
Comment (bziondik) [inline controller/QtiCreator.php:306]: Why you need to remove plugin? It should't be register

**Review by bziondik — APPROVED** [review]
Can you add unit tests for new plugin?
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: controller/QtiCreator.php [hunk 2]
```diff
@@ -300,6 +300,10 @@ protected function getCreatorConfig(core_kernel_classes_Resource $item)
             $config->addPlugin('blockAdder', 'taoQtiItem/qtiCreator/plugins/content/blockAdder', 'content');
         }
 
+        if ($config->getProperty('scrollable-multi-column') == true) {
```
Comment (bartlomiejmarszal) [inline controller/QtiCreator.php:303]: Should we apply here strict comparison?
```suggestion
        if ($config->getProperty('scrollable-multi-column') === true) {
```

**Review by bartlomiejmarszal — COMMENTED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bartlomiejmarszal — APPROVED** [review]
LGTM

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-anastasia.playground.kitchen.it.taocloud.org:44451/

**Pull request merged**